target/linux/ramips/image/mt7621.mk

```diff
@@ -1068,6 +1068,20 @@ define Device/totolink_x5000r
 endef
 TARGET_DEVICES += totolink_x5000r
 
+define Device/tplink_archer-c6u-v1
+  $(Device/dsa-migration)
+  $(Device/tplink-safeloader)
+  IMAGE_SIZE := 15744k
+  DEVICE_MODEL := ARCHER-C6U
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e \
+	kmod-mt7663-firmware-ap kmod-mt7663-firmware-sta \
```

Comment:
This is not used for any device in the entire tree. Do we really need it?

Reply:
Yes, mt7613 requires `kmod-mt7663-firmware-ap` or `kmod-mt7663-firmware-sta`. The mt76 driver (mt7615) attempts to load the *-sta binaries first and if they are missing loads the *-ap ones.